USERGUIDE.md

```diff
@@ -0,0 +1,113 @@
+# User Guide
+
+This contains typical usage examples and workflow for users using this plugin.
+
+---
+
+## Prepping new design files
+
+
+### Adding Print and Cut Layers
+
+You are recommended to add a layer named `Print` and `Cut`.
+The `Print` layer in addition to `Regmarks` will be ignored by `Send to Silhouette` extention.
+
+
+### Add registration mark
+
+<img src="./assets/screenshot_of_the_regmarked_document.png" alt="screenshot of the regmarked document" height=200px>
+
+The plotter will search the registration marks at the given positions.
+If it locates the marks, they will serve as accurate reference and define the origin.
+Therefore it is necessary to set the correct offset values of the mark.
+As a result the cut will go precisely along the graphics.
+
+1. Extention > Render > Silhouette Regmarks
+1. Check regmark from document left and top is set to desired value
+1. Set mark to mark distance or clear it to zero if autocalculating from document size
+1. Press Apply
+
+This will create a new layer called `Regmarks` with the newly generated registration mark.
+
+On the bottom will also be a string shown below that would remind you what settings this was generated with.
+- `mark distance from document: Left=10.0mm, Top=10.0mm; mark to mark distance: X=190.0mm, Y=277.0mm;`
+
+Note: You have the option of using the provided template at `examples/registration-marks-cameo-silhouette-a4-maxi.svg` for Silhouette Cameo using A4 paper format.
+
+
+---
+
+## Plot
+
+<img src="./assets/screenshot_of_send_to_silhouette.png" alt="screenshot of Send to Silhouette" height=200px>
+
+1. Open your document with inkscape.
+2. Ensure the unit of document width and height is mm or inch, but not px. (File - Document settings - Page - Custom - Unit mm)
```

Comment:
this line is no longer needed - you may drop it and rearrange the numbering...
2. Ensure the unit of document width and height is mm or inch, but not px. (File - Document settings - Page - Custom - Unit mm)